packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/EditQueueModal.tsx

```diff
@@ -296,12 +287,16 @@ const EditQueueModal: React.FC<EditQueueModalProps> = ({
             })
         : Promise.resolve()
     const updateQueueParams: UpdateQueueParams = pick(values, [
+      'type',
       'notes',
       'allowQuestions',
+      'zoomLink',
```

Comment:
I'm noticing that I can't clear the zoomLink, it seems to still call the edit course endpoint?

![image](https://github.com/user-attachments/assets/28948014-40d0-4a2d-ae41-708c4b20fd31)
![image](https://github.com/user-attachments/assets/b5da99f9-b732-4cf3-ac04-f0a3b2886342)